docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
 
 !!! tip
 
-    For the best reading experience, it is recommended that you read through this section.
+    It is recommended that you go through this section to get the best reading experience.
 
-## Conventions Of Style
+## Conventions Of Writing Style
 
-- Those labeled `*` after the title are optional chapters with relatively difficult content. If you have limited time, it is advisable to skip them.
-- Proper nouns and words and phrases with specific meanings are marked with `"double quotes"` to avoid ambiguity.
-- Important proper nouns and their English translations are marked with `" "` in parentheses, e.g. `"array array"` . It is recommended to memorize them for reading the literature.
-- **Bolded text** Indicates key content or summary statements, which deserve special attention.
-- When it comes to terms that are inconsistent between programming languages, this book follows Python, for example using $\text{None}$ to mean "empty".
-- This book partially abandons the specification of annotations in programming languages in exchange for a more compact layout of the content. There are three main types of annotations: title annotations, content annotations, and multi-line annotations.
+- Chapters marked with an asterisk * in their titles are optional and more challenging. If you are short on time, it is advisable to skip them.
+- Key technical terms and their English equivalents are enclosed in `「 」` brackets, for example, `「array」`. It's advisable to familiarize yourself with these for better comprehension of technical texts.
+- Proprietary terms and words with specific meanings are indicated with “quotation marks” to avoid ambiguity.
+- **Bolded text** indicates key content or summary statements, which deserve special attention.
+- When it comes to terms that are inconsistent between programming languages, this book follows Python, for example using $\text{None}$ to mean "null".
+- This book partially abandons the specification of annotations in programming languages in exchange for a more compact layout of the content. Comments are categorized into three types: title comments, content comments, and multiline comments.
```

Comment:
it can also be written as (optional):
"- This book opts for a more concise content layout by partly abandoning the specification of annotations in programming languages. The comments primarily consist of three types: title comments, content comments, and multi-line comments."